modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/IgniteMethod.java

```diff
@@ -98,17 +98,13 @@ public enum IgniteMethod {
     IS_NOT_DISTINCT_FROM(Objects.class, "equals", Object.class, Object.class),
 
     /** See {@link UUID#randomUUID()}. */
-    RAND_UUID(UUID.class, "randomUUID"),
+    RAND_UUID(IgniteSqlFunctions.class, "randUuid"),
 
     LENGTH(IgniteSqlFunctions.class, "length", Object.class),
 
     OCTET_LENGTH(IgniteSqlFunctions.class, "octetLength", ByteString.class),
     OCTET_LENGTH2(IgniteSqlFunctions.class, "octetLength", String.class),
 
-    /** See {@link IgniteSqlFunctions#genRandomUuid()}. */
-    // TODO This function should removed when https://issues.apache.org/jira/browse/IGNITE-19103 is complete.
-    GEN_RANDOM_UUID(IgniteSqlFunctions.class, "genRandomUuid"),
```

Comment:
please remove the function body from `IgniteSqlFunctions` class as well.

Reply:
Fixed.